src/execution/riscv64/simulator-riscv64.h

```diff
@@ -574,9 +574,10 @@ class Simulator : public SimulatorBase {
   struct Breakpoint {
     Instruction* location;
     bool enabled;
+    bool temporary;
```

Comment:
`temporary` may be mistaken as temp variable, use a more intuitive name, e.g., `is_tbreak`.